File: tma-task-vue/src/Data/ItemResponse.js
class ItemResponse {
    constructor(ItemID, ItemGroup, UnitOfMeasurement, Quantity, PriceWithoutVAT, Status, StorageLocation, ContactPerson, Photo) {
        this.ItemID = ItemID;
        this.ItemGroup = ItemGroup;
        this.UnitOfMeasurement = UnitOfMeasurement;
        this.Quantity = Quantity;
        this.PriceWithoutVAT = PriceWithoutVAT;
        this.Status = Status;
        this.StorageLocation = StorageLocation;
        this.ContactPerson = ContactPerson;
        this.Photo = Photo;
    }
}

export default ItemResponse;

File: TMAWarehouseAPI/TMAWarehouseAPI/Models/DTO/ItemResponse.cs
namespace TMAWarehouseAPI.Models.DTO
{
    public class ItemResponse
    {
        public int ItemID { get; set; }
        public string ItemGroup { get; set; }
        public string UnitOfMeasurement { get; set; }
        public int Quantity { get; set; }
        public decimal PriceWithoutVAT { get; set; }
        public string Status { get; set; }
        public string StorageLocation { get; set; }
        public string ContactPerson { get; set; }
        public string Photo { get; set; }
    }
}
